lib/opensusebasetest.pm

```diff
@@ -552,9 +553,12 @@ sub post_fail_hook {
             record_info("no $program", "Could not find '$program' on the system", result => 'fail') && die "$program does not exist on the system";
         }
     }
-    return unless $self->{in_wait_boot};
-    if (wait_serial qr/Reached target Shutdown/) {
-        record_info 'shutdown', 'At least we reached target Shutdown';
+    return unless ($self->{in_wait_boot} || $self->{in_boot_desktop});
+    if ($self->{in_wait_boot}) {
+        record_info('shutdown', 'At least we reached target Shutdown') if (wait_serial 'Reached target Shutdown');
```

Comment:
Do we call this code on remote backends? Because for zVM we cannot use wait_serial, as we have `expect_3270`

Reply:
So it won't work on zVM, as we also call `wait_boot` which sets `in_wait_boot=1` , but we could not find any test which is not install only on zVM. Also, it's not that critical as it's called in case of failure, but can be indeed misleading as we will get timed out wait_serial.